fontforge/python.c

```diff
@@ -12003,7 +12003,8 @@ return(-1);
 return( -1 );
     sizes = malloc((cnt+1)*sizeof(int));
     for ( i=0; i<cnt; ++i ) {
-	if ( !PyArg_ParseTuple(PyTuple_GetItem(value,i),"i", &sizes[i])) {
+	sizes[i] = PyLong_AsLong(PyTuple_GetItem(value,i));
```

Comment:
This fixes the issue, but will break the existing scripts for those who managed to figure out this peculiarity.

On one hand, preserving existing state has merits, but on the other hand we do break things from time to time, like when migrating to Python3 or updating AGL for new fonts.

@skef, what do you think?

Reply:
Yeah, I'm fine with standardizing on flat ints but the code should still accept the singleton tuples in case scripts have relied on that behavior. It's not that hard to code that way.